test/config/utility.cc

```diff
@@ -0,0 +1,98 @@
+#include "test/config/utility.h"
+
+#include "common/common/assert.h"
+#include "common/protobuf/utility.h"
+
+#include "test/test_common/environment.h"
+#include "test/test_common/network_utility.h"
+
+#include "api/filter/http_connection_manager.pb.h"
+
+namespace Envoy {
+
+std::string basic_config = R"EOF(
+{
+  "static_resources": {
+    "listeners":[{
+      "name":"listener_0",
```

Comment:
Why don't we have an `addListener()` that builds this? That way we can easily add multiple listeners.

Reply:
I'd like that, but this TODO came from Matt who preferred the basic config in one place.

I figure we'll start with this, add the utilities as tests tweak config, and then maybe get used to just using the helpers?  Or we'll leave this here as a sample config.

Reply:
SGTM, don't want to boil the ocean ;)

Reply:
`addListener()` is OK with me, presumably with the listener template being defined somewhere. Really all I'm trying to maintain is some level of readability of what is going on in each test. As long as we have the right helpers and the right snippets to look back on it's probably fine.